lavamoat/build-system/policy.json

```diff
@@ -2976,6 +2921,13 @@
         "debounce-stream>through": true
       }
     },
+    "@lavamoat/lavapack>readable-stream>abort-controller>event-target-shim": {
+      "globals": {
+        "Event": true,
+        "EventTarget": true,
+        "console": true
```

Comment:
We could restrict these in overrides later as I don't think this shim has any work left to do.